src/test/rocket.test.js

```diff
@@ -0,0 +1,23 @@
+import { render, cleanup } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import store from '../redux/store';
+import Rocket from '../components/Rocket';
+
+afterEach(cleanup);
+describe('Testing rocket component', () => {
+  test('Generating the snapshots for rocket component', () => {
+    const rocket = {
+      id: '123',
+      rocket_name: 'Test Rocket',
+      flickr_images: ['image1.jpg', 'image2.jpg'],
+      description: 'Test rocket description',
+      reserved: false,
+    };
+    const { container } = render(
+      <Provider store={store}>
+        <Rocket rocket={rocket} />
+      </Provider>,
+    );
+    expect(container).toMatchSnapshot();
+  });
```

Comment:
- [ ] Hello team, good job overall. For the test, I have seen you have done a `Snapshot` test. this is great and important but we need to write unit tests with React Testing Library and Jest (for unit tests) alongside the snapshot testes.

Reply:
Resolved